addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/WebSocketMessageDTO.java

```diff
@@ -143,25 +143,25 @@ public String toString() {
      * @param other
      */
     public void copyInto(WebSocketMessageDTO other) {
-        other.channel = this.channel;
```

Comment:
This is unnecessary.

Reply:
I think that was an automated change that SonatLint made, will revert.